pkg/activator/handler/concurrency_reporter_test.go

```diff
@@ -382,12 +382,8 @@ func TestMetricsReported(t *testing.T) {
 	<-s.statChan // The actual report we want to see
 
 	wantTags := map[string]string{
-		metricskey.LabelRevisionName:      rev1.Name,
-		metricskey.LabelNamespaceName:     rev1.Namespace,
-		metricskey.LabelServiceName:       "service-" + rev1.Name,
-		metricskey.LabelConfigurationName: "config-" + rev1.Name,
-		metricskey.PodName:                "the-best-activator",
-		metricskey.ContainerName:          "activator",
```

Comment:
Can we test the resource stuff instead then? Seems like we're strictly removing test coverage here.

Reply:
I'm going to see about updating metricstest to provide some tools for working with Resourced metrics.

Reply:
Done.  Thanks to @evankanderson's work in pkg/metrics, we can now verify resource now.